Update docs for quality metrics (#138)
datajoint/element-array-ephys: docs/src/concepts.md

| --- | --- |
| ProbeLevelReport | A table to store drift map figures generated from each recording probe. |
| UnitLevelReport | A table to store figures (waveforms, autocorrelogram, peak waveform + neighbors) generated for each unit. |
| QualityMetricCutoffs | A table to store cut-off values for cluster quality metrics. |

> | QualityMetricSet | A table to store a set of quality metric values for clusters and its cut-offs. |

Reviewer: ```suggestion
| QualityMetricSet | A manual table to match a set of quality metric values for clusters with desired cut-offs. |
```
